docs/docs/api-reference/core/useSetRecoilState.md

```diff
@@ -7,6 +7,37 @@ Returns a setter function for updating the value of writeable Recoil state.
 
 ---
 
-- `state`: writeable Recoil state.
+- `state`: writeable Recoil state (an [`atom`](/docs/api-reference/core/atom) or a _writeable_ [`selector`](/docs/api-reference/core/selector))
+
+This is the recommended hook to use when a component intends to write to state without reading it.
```

Comment:
"This is the recommended hook to use when a component intends to write to state without reading it.  If a component used the `useRecoilState()` hook to get the setter, it would also subscribe to updates and re-render when the atom or selector updated.  Using `useSetRecoilState()` allows a component to set the value without re-rendering when the value changes."

Reply:
Cool! Thanks for the more detail description :)